### PR TITLE
Add Cloud Connect

### PR DESCRIPTION
**Pull request by jblashka**
Adding Cloud Connect. This is the react app responsible for talking to the IT backend during the Cloud Marketplace customer purchase onboarding process

### CODE REVIEW

File: chrome/fed-modules.json [hunk 1]
```diff
@@ -699,5 +699,17 @@
               ]
           }
       ]
+    },
+    "connect": {
+        "manifestLocation": "/apps/connect/fed-mods.json",
```
Comment (Hyperkid123) [inline chrome/fed-modules.json]: The manifest location depends on your `insights.appname` from your `package.json`

So it should be 

```
"manifestLocation": "/apps/statosphere-plf/fed-mods.json",
```

**Review by karelhala — COMMENTED** [review]
This is effectively a new bundle, isn't it? You'll need a new navigation file as well called `connect-navigation.json`.

```JSON
{
    "id": "connect",
    "title": "Cloud connect",
    "navItems": [
        {
            "appId": "connect",
            "title": "Connect",
            "filterable": false,
            "href": "/connect"
        },
   ]
}
```

**Comment by jblashka** [comment]
This is a bit of an odd one because there's intentionally no way to navigate to this app. It's only used during the purchase flow from AWS & Azure; if a user isn't coming directly from those Cloud Marketplaces via a special link generated within the Cloud Marketplaces, we can't do anything.

File: chrome/fed-modules.json [hunk 2]
```diff
@@ -699,5 +699,17 @@
               ]
           }
       ]
+    },
+    "connect": {
```
Comment (Hyperkid123) [inline chrome/fed-modules.json]: The key should be `statospherePlf`. Again it depends on the appname, but due to some browser env limitations we can't support the dash character `-`. That is why its converted to camelCase

**Comment by Hyperkid123** [comment]
> This is effectively a new bundle, isn't it? You'll need a new navigation file as well called `connect-navigation.json`.
> 
> ```json
> {
>     "id": "connect",
>     "title": "Cloud connect",
>     "navItems": [
>         {
>             "appId": "connect",
>             "title": "Connect",
>             "filterable": false,
>             "href": "/connect"
>         },
>    ]
> }
> ```

@karelhala this bundle will not have left navigation. It will also need a bit more different layout.

File: chrome/fed-modules.json (hunk 1 above)
Comment (jblashka) [reply chrome/fed-modules.json]: I was going to update that insights.appname to connect as well

Comment (Hyperkid123) [reply chrome/fed-modules.json]: Ah, then it's OK. And you can use the `connect` keys/path

**Review by Hyperkid123 — APPROVED** [review]

**Pull request closed without merging**